keys/node.keys

```diff
@@ -11,3 +11,4 @@ DD8F2338BAE7501E3DD5AC78C273792F7D83545D
 A48C2BEE680E841632CD4E44F07496B3EB3C1762
 108F52B48DB57BB0CC439B2997B01419BD92F80A
 B9E2F5981AA6E0CD28160D9FF13993A75599653C
+890C08DB8579162FEE0DF9DB8BEAB4DFCF555EF4
```

Comment:
nit: Think this is supposed the match the order from the README file. We have a script that pull it in that order